src/framework/theme/components/menu/menu.component.scss

```diff
@@ -26,6 +26,10 @@
     }
   }
 
+  .menu-item nb-badge {
+    position: inherit;
```

Comment:
Why `inherit`? Do you want to change position from `absolute`? If so, let's use `static` value, it's default element position.